### PR TITLE
add conv_ai

### PR DESCRIPTION
**Pull request by patil-suraj**
Adding ConvAI dataset https://github.com/DeepPavlov/convai/tree/master/2017

### CODE REVIEW

File: datasets/conv_ai/conv_ai.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import datasets
+
+
+_DESCRIPTION = """\
+ConvAI is a dataset of human-to-bot conversations labelled for quality. \
+This data can be used to train a metric for evaluating dialogue systems. \
+Moreover, it can be used in the development of chatbots themselves: it contains the information \
+on the quality of utterances and entire dialogues, that can guide a dialogue system in search of better answers.
+"""
+
```
Comment (lhoestq) [inline datasets/conv_ai/conv_ai.py:30]: no citation ?

**Review by lhoestq — APPROVED** [review]
LGTM !

feel free to merge if you have no other changes to make

**Comment by julien-c** [comment]
This one will make @thomwolf reminisce ;)

File: datasets/conv_ai/conv_ai.py (hunk 1 above)
Comment (patil-suraj) [reply datasets/conv_ai/conv_ai.py:30]: couldn't find any citation info, the dataset and paper is hosted on github

**Comment by patil-suraj** [comment]
Merging.

**Pull request merged**